### DISCUSSION TITLE
Discord changes the oAuth page and it doesn't match the guide

### DISCUSSION THREAD

**Discussion by mezanddav** [jagrosh/MusicBot#1484] [Questions] [answered]
Hey,

The addon works great, I am self hosting it on my own server, running in the background on 2 servers. At least that's what I wanted but since they changed the oAuth page design I am unsure what permissions are required. 

Incorrect guide link: https://jmusicbot.com/adding-your-bot/#troubleshooting

![image](https://github.com/jagrosh/MusicBot/assets/33984916/61342726-1345-4f54-860f-211096f37ccc)

What should be checked here?

Suggestion: Update docs with the latest Discord UI changes.

**Comment by MichailiK** [comment, marked as answer]
You don't *need to* pick any permissions. JMusicBot only needs pretty basic permissions, so chances are pretty high the @everyone role has these permissions already.
